workshops/request.html

```diff
@@ -39,189 +53,8 @@
-<span class="ss-choice-label">For-profit: US$10,000</span>
-</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Self-organized (no fee - please let us know if you wish to make a donation)" id="group_793641257_6" role="radio" class="ss-q-radio" aria-label="Self-organized (no fee - please let us know if you wish to make a donation)" required="" aria-required="true"></span>
-<span class="ss-choice-label">Self-organized (no fee - please let us know if you wish to make a donation)</span>
-</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Waiver requested (please give details in &quot;Anything else&quot;)" id="group_793641257_7" role="radio" class="ss-q-radio" aria-label="Waiver requested (please give details in &quot;Anything else&quot;)" required="" aria-required="true"></span>
-<span class="ss-choice-label">Waiver requested (please give details in "Anything else")</span>
-</label></li></ul>
-<div class="error-message" id="185056084_errorMessage"></div>
-<div class="required-message">This is a required question</div></div></div></div> <div class="ss-form-question errorbox-good" role="listitem">
-<div dir="ltr" class="ss-item  ss-paragraph-text"><div class="ss-form-entry">
-<label class="ss-q-item-label" for="entry_185044375"><div class="ss-q-title">Anything else?
-</div>
-<div class="ss-q-help ss-secondary-text" dir="ltr">What else do you want us to know about your workshop? About your attendees? About you?</div></label>
-<textarea name="entry.185044375" rows="8" cols="0" class="ss-q-long" id="entry_185044375" dir="auto" aria-label="Anything else? What else do you want us to know about your workshop? About your attendees? About you? "></textarea>
-<div class="error-message" id="1163896537_errorMessage"></div>
-<div class="required-message">This is a required question</div>
-</div></div></div>
-<input type="hidden" name="draftResponse" value="[,,&quot;-4405048771215045936&quot;]
-">
-<input type="hidden" name="pageHistory" value="0">
-<input type="hidden" name="fbzx" value="-4405048771215045936">
-<div class="ss-item ss-navigate"><table id="navigation-table"><tbody><tr><td class="ss-form-entry goog-inline-block" id="navigation-buttons" dir="ltr">
-<input type="submit" name="submit" value="Submit" id="ss-submit" class="jfk-button jfk-button-action ">
-<div class="ss-password-warning ss-secondary-text">Never submit passwords through Google Forms.</div></td>
-</tr></tbody></table></div></ol></form></div>
+<p>
+  We strive to be a global project and support diversity in science.
+  If you wish to offer a workshop that would further these goals,
+  please <a href="{{site.contact}}">contact us</a> regarding a waiver for the administration fee.
```

Comment:
I would add link to the administrative fee page here, but that's not a critical issue in this PR.